### PR TITLE
add config section, misc

### PR DESCRIPTION
**Pull request by lisakowen**
added a config section.  also made some minor wording edits.

noticed some control-m s in the diff.  not sure where those are coming from ... don't seem to affect the html generation.

### CODE REVIEW

**Comment by janebeckman** [comment]
Custom Configuration looks good.

File: markdown/ranger/ranger-integration-config.html.md.erb [hunk 1]
```diff
@@ -124,8 +116,39 @@ Once the connection between HAWQ and Ranger is configured, you can either set up
...
+
+Configuration files for the HAWQ Ranger Plug-in Service are located in the `$GPHOME/ranger/etc` directory. These files include:
+
+| File     |  Description     |
+|-------------|---------------------------|
+| ranger-hawq-audit.xml |  HAWQ Ranger audit-related configuration, including the audit provider (log4j, Solr, HDFS) and provider-specific configuration |
+| ranger-hawq-security.xml |  HAWQ Ranger service configuration, including the policy change polling interval |
+| rps.properties |  HAWQ Ranger deployment-related configuration, including the RPS port definition and JVM parameters|
+
+You should not need to modify RPS configuration in a default installation.  Any configuration changes you do make after you have registered the HAWQ Ranger Plug-in requires a restart of the service. You can either restart the HAWQ cluster or restart just the HAWQ Ranger Plug-in Service:
```
Comment (dyozie) [inline markdown/ranger/ranger-integration-config.html.md.erb]: Get rid of RPS acronym.  I'm also not sure that we want to say "you should not need to modify..."  Auditing, polling interval seem like pretty common changes that might be made.

File: markdown/ranger/ranger-integration-config.html.md.erb [hunk 2]
```diff
@@ -124,8 +116,39 @@ Once the connection between HAWQ and Ranger is configured, you can either set up
...
+``` shell
+gpadmin@master$ /usr/local/hawq/ranger/bin/rps.sh stop
+gpadmin@master$ /usr/local/hawq/ranger/bin/rps.sh start
+```
+
+### <a id="caching"></a>Changing the Frequency of Policy Caching
  
-You may wish to change the frequency of policy caching to suit your individual needs.
\ No newline at end of file
+The default polling interval for HAWQ RPS policy updates is 30 seconds. To increase or decrease this value, update the `ranger.plugin.hawq.policy.pollIntervalMs` property setting in the `ranger-hawq-security.xml` file:
```
Comment (dyozie) [inline markdown/ranger/ranger-integration-config.html.md.erb]: Nix the RPS again.

Comment (lisakowen) [inline markdown/ranger/ranger-integration-config.html.md.erb]: incorporated your updates, david.

**Comment by lisakowen** [comment]
PR merged; closing

**Pull request closed without merging**